web/src/components/UpgradeBrowser.js

```diff
@@ -11,12 +11,6 @@ const browsers = [
     name: 'Mozilla Firefox',
     link: 'https://www.mozilla.org/en-US/firefox/new/'
   },
-  {
-    id: 'ie',
-    name: 'Internet Explorer 11',
-    link:
-      'https://support.microsoft.com/en-us/help/17621/internet-explorer-downloads'
-  },
```

Comment:
Should we create an issue to add an Edge download link when the Chromium builds are released? I think they're supposed to be out in the first quarter of 2020, so not that long.